app/controllers/artefacts_controller.rb

```diff
@@ -0,0 +1,55 @@
+class ArtefactsController < ApplicationController
+  def new
+    @artefact = Artefact.new(content_id: SecureRandom.uuid)
+  end
+
+  def create
+    @artefact = Artefact.new
+    if @artefact.update_attributes_as(current_user, creatable_params)
+      edition = create_edition(@artefact)
```

Comment:
Some thoughts on this. Previously in Panopticon, after having created the artefact the controller would redirect to "/admin/publications/#{artefact.id}" in Publisher. The [corresponding controller method](https://github.com/alphagov/publisher/blob/master/app/controllers/publications_controller.rb#L2-L10) has some logic which does a `find_or_create` on the Edition, so the creation of the edition was handled in the PublicationsController. Do you have any strong opinions on keeping that behaviour? (mostly asking because it touches our other PR).